src/tests/system/tests/test_kcm.py

```diff
@@ -507,6 +507,8 @@ def test_kcm_ssh_login_creates_kerberos_ticket(client: Client, ldap: LDAP, kdc:
     kdc.principal("user1").add()
 
     client.sssd.common.krb5_auth(kdc)
+    client.sssd.domain["krb5_realm"] = "TEST"
+    client.sssd.domain["krb5_server"] = "kdc.test"
```

Comment:
Instead of hard-coding realm and server's value, can we add variable, if possible?

Reply:
Changed.

Reply:
Executed in gating and it works:
2024-04-04T12:07:08 ../sssd/src/tests/system/tests/test_kcm.py::test_kcm_ssh_login_creates_kerberos_ticket (ldap) PASSED [ 58%]

Reply:
Marking as resolved.